08LessonEight/homework/exercise-06.html

```diff
@@ -1,10 +1,30 @@
 <!DOCTYPE html>
 <html>
-
+<link rel="stylesheet" href="css/exercise-06-style.css" />
 <head>
     <title>Exercise 06</title>
 </head>
+<body>
+  <section class="container">
+    <section class="box1">
+      <img src="img/truck.jpg"/>
+      <h1>SAFE & SECURE DELIVERY</h1>
+      <p>my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles.</p>
+      <span>Read More</span>
+    </section>
+    <section class="box2">
+      <img src="img/plane.jpg"/>
+      <h1>SAFE & SECURE DELIVERY</h1>
+      <p>my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles.</p>
+      <span>Read More</span>
+    </section>
+    <section class="truck">
+      <img src="img/train.jpg"/>
+      <h1>SAFE & SECURE DELIVERY</h1>
+      <p>my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles. my name is Julio and I like turtles.</p>
+      <span>Read More</span>
+    </section>
 
-<body></body>
-
-</html>
\ No newline at end of file
+  </section>
+</body>
+</html>

```

Comment:
Great job on this assignment Julio. The only thing I would say to work on would be buttons. They also need to line up horizontally. Keep up the good work!